src/components/tools/table-tool/table-header-menu.tsx

```diff
@@ -79,7 +79,7 @@ export class TableHeaderMenu extends React.Component<IProps, IState> {
     });
   }
 
-  public buildColumnNameErrorGetter(columnHasExpression: boolean): (name: string) => string | undefined {
+  public buildColumnNameValidator(columnHasExpression: boolean): (name: string) => string | undefined {
```

Comment:
Clever -- I like it. Another approach would have been to have the nameValidator() function take an additional boolean parameter and then use bind() to bind the boolean argument before passing it down, but using a closure this way works just as well.